File: modulo03/bloco22/dia02/exercicios/exercicio01.js
function calcula(num1, num2, num3) {
  return new Promise((resolve, reject) =>{
    if(typeof num1 != 'number' || typeof num2 != 'number' || typeof num3 != 'number') {
        return reject(new Error("Informe apenas números"));
      };

    const result = ((num1 + num2) * num3);

    result < 50
      ? reject(new Error("Valor muito baixo"))
      : resolve(result);
    });
};

calcula(10, 20, 30)
  .then((resolve) => console.log(resolve))
  .catch((error) => console.log(error.message));

calcula('a', 10, 20)
  .then((resolve) => console.log(resolve))
  .catch((error) => console.log(error.message));

File: modulo04/bloco32/dia01/exercicios/exercicio01.py
def maior_numero(num1, num2):
    if num1 > num2:
        return num1
    else:
        return num2


def maior_numero_2(num1, num2):
    return max(num1, num2)


print(maior_numero(15, 38))
print(maior_numero(45, 700))
print(maior_numero(25, 2))
print(maior_numero(189, 1568))

print(maior_numero_2(15, 38))
print(maior_numero_2(45, 700))
print(maior_numero_2(25, 2))
print(maior_numero_2(189, 1568))
